packaging/packager

```diff
@@ -56,26 +55,41 @@ if ! type zip > /dev/null 2>&1; then
     echo "zip utility is not found. Please install it and re-run this script"
     exit 1
 fi
-function package_libc_via_pacman {
+
+function pluck_so_files() {
+    sed -E '/\.so$|\.so\.[0-9]+$/!d'
+}
+
+function package_libc_alpine() {
+    if grep -F "Alpine Linux" < /etc/os-release > /dev/null; then
+        if type apk > /dev/null 2>&1; then
+            apk info -L musl 2>/dev/null | pluck_so_files | sed 's/^/\//'
```

Comment:
Same here if applicable. If not add a comment.

Reply:
No problem.